### PR TITLE
Code Clean Up

### PR DESCRIPTION
**Pull request by kingdomlevel**
This PR's main purpose is to tidy up the code and put us in a better place moving forwards with development of the react app.

In this feature:

- removes jQuery as a dependency
- removes bootstrap as a dependency and references to it in the JSX code
- removes almost all styling to give us a "blank slate" to work from in working towards Ananda's design
- brings in the start of react-router-dom (not intended behaviour, but it was a place to start from today!)

## AC

- [ ] The app renders landing page content in the browse
- [ ] The app renders a nav bar with clickable links
- [ ] Clickable links change the browser's URL path to the intended place (content rendering on screen won't change at this stage)

### CODE REVIEW

**Review by endyranaudo — APPROVED** [review]
All good and clear

File: src/App.js [hunk 1]
```diff
@@ -1,81 +1,71 @@
...
+import { BrowserRouter as Router } from 'react-router-dom';
+import Header from './components/header/Header';
+import Footer from './components/footer/Footer';
+import SideMenu from './components/sidemenu/SideMenu';
 import './App.css';
+import { fetchLandingPageContent, fetchMainMenuItems, fetchSubMenuItems } from './helpers/ContentConsumer';
 
 function App() {
-
-  const BASE_URL = 'http://54.78.155.180:1337'
```
Comment (yemijaiyeola) [inline src/App.js:12]: let's set as env var

**Review by yemijaiyeola — CHANGES_REQUESTED** [review]
Thanks for this.  Looks good.  Only one comment to fix the BASE_URL text.  

Needs refactoring just tidy up but we can do that as a separate PR.  If we just fix the env var then we can push through.

**Review by yemijaiyeola — COMMENTED** [review]
add tests

**Comment by yemijaiyeola** [comment]
@kingdomlevel  .I have used this as the basis for the basic-structure and footer pr.  Will continue to evolve. 💯

**Pull request merged**